controllers/sessionController.js

```diff
@@ -0,0 +1,48 @@
+const User = require("../models/User");
+const parseVErr = require("../util/parseValidationErr");
+
+const registerShow = (req, res) => {
+  res.render("register");
+};
+
+const registerDo = async (req, res, next) => {
+  if (req.body.password != req.body.password1) {
```

Comment:
When trying to use this, I keep getting `Cannot read properties of undefined (reading 'password')` Take a look and see where this could be causing an error.